### PR TITLE
Feature/ppna 3983 new UI

### PR DESCRIPTION
**Pull request by JacekClearcode**

### CODE REVIEW

**Review by RomanSek — APPROVED** [review]

File: _static/css/custom_1626336655663.css [hunk 1]
```diff
@@ -238,19 +232,202 @@ body {
...
+
+#redoc-container [data-section-id*="/paths/"] {
+    padding-bottom: 20px !important;
+}
+
+#redoc-container h5 {
+    box-sizing: border-box;
+    border: 1px solid #D5D9E0;
+    background-color: #FFFFFF;
+    padding: 16PX;
```
Comment (Pyszo) [inline _static/css/custom_1626336655663.css:297]: don't shout at me :V ( PX => px )

File: _static/css/custom_1626336655663.css [hunk 2]
```diff
@@ -238,19 +232,202 @@ body {
...
+}
+
+#redoc-container .dtUibw, #redoc-container .fLUKgj {
```
Comment (Pyszo) [inline _static/css/custom_1626336655663.css:300]: I believe that those "dtUibw" etc. aren't just a random hashes that will change every redoc version, right? :D

File: _static/css/custom_1626336655663.css [hunk 3]
```diff
@@ -697,4 +871,575 @@ a.link-large {
...
+#redoc-container .menu-content {
+    display: none;
+}
+
+#redoc-container .api-content {
+    width: 100%;
+}
+
+#redoc-container .sc-iuJeZd.fLUKgj {
+    width: calc(100% * 0.4);
```
Comment (Pyszo) [inline _static/css/custom_1626336655663.css:1037]: 40% ? :D

File: _static/images/menu.svg [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Menu</title>
```
Comment (Pyszo) [inline _static/images/menu.svg:3]: it may show tooltip, is it intendet?

**Review by Pyszo — APPROVED** [review]

**Pull request merged**